go.mod

```diff
@@ -260,6 +252,7 @@ replace github.com/cosmos/solidity-ibc-eureka/abigen => ./solidity-ibc-eureka/ab
 replace (
 	github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
 	github.com/ChainSafe/go-schnorrkel/1 => github.com/ChainSafe/go-schnorrkel v1.0.0
+	github.com/consensys/gnark => github.com/jtguibas/gnark v0.0.0-20240923234830-41125bc1909c
```

Comment:
[question] why do we need to use a fork of gnark?

Reply:
sp1 uses this fork. I don't think gnark needs to be around anymore since we don't really use it directly in this repo